datahub-web-react/src/app/entity/shared/containers/profile/utils.ts

```diff
@@ -221,3 +221,6 @@ export function sortEntityProfileTabs(appConfig: AppConfig, entityType: EntityTy
 
     return sortedTabs;
 }
+
+// To focus out from modal popup so autofocus works
+export const getContainer = () => document.getElementById('root') as HTMLElement;
```

Comment:
again, did we try to add the id to the app.tsx? 

let's move this into a different place. let's put it in a root folder: src/app/utils/focus

Reply:
let's say getRootModalContainer. 

Also please write a bit of a more detailed comment here.

Reply:
can we be 100% positive that the "root" id will never change?

Reply:
`ReactDOM.render(
    <React.StrictMode>
        <App />
    </React.StrictMode>,
    document.getElementById('root'),
);`
here also we are using root as id if this cannot be change we are 100% sure id will not be change

Reply:
> let's move this into a different place. let's put it in a root folder: src/app/utils/focus
sure will add this to app/utils/focus and change name to `getModalDomContainer`

Reply:
> again, did we try to add the id to the app.tsx?

In App.tsx we have written all context and routing related code, for adding Id we have to add extra node of div and other person will be confused  why we have given id there instead we can use root as id that will not change

Reply:
update the MR please review and let me know